pyccl/core.py

```diff
@@ -419,12 +424,18 @@ def _build_config(
     def _build_parameters(
             self, Omega_c=None, Omega_b=None, h=None, n_s=None, sigma8=None,
             A_s=None, Omega_k=None, Neff=None, m_nu=None, m_nu_type=None,
-            w0=None, wa=None, T_CMB=None,
+            w0=None, wa=None, T_CMB=None, T_ncdm=None,
             bcm_log10Mc=None, bcm_etab=None, bcm_ks=None,
             mu_0=None, sigma_0=None, c1_mg=None, c2_mg=None, lambda_mg=None,
             z_mg=None, df_mg=None, Omega_g=None,
             extra_parameters=None):
         """Build a ccl_parameters struct"""
+        # Fill-in defaults (SWIG converts `numpy.nan` to `NAN`)
+        A_s = np.nan if A_s is None else A_s
+        sigma8 = np.nan if sigma8 is None else sigma8
+        T_CMB = const.T_CMB if T_CMB is None else T_CMB
+        Omega_g = np.nan if Omega_g is None else Omega_g
+        T_ncdm = const.T_ncdm if T_ncdm is None else T_ncdm
```

Comment:
ditto here. Let's write the values here directly. It's the only place in the code where they need to appear

Reply:
Try that and see what happens, a wormhole will open.